functions/Read-DbaTransactionLog.ps1

```diff
@@ -94,7 +94,12 @@ Will read the contents of the transaction log of MyDatabase on SQL Server Instan
 		else
 		{
 			#Warn if more than 0.5GB of live log. Dodgy conversion as SMO returns the value in an unhelpful format :(
-			if ($server.databases[$Database].LogFiles.usedspace/1000 -ge 500) # this will cause enumeration and needs to be addressed
```

Comment:
These small fixes look good.
But I'm wondering why there's an END block around this code, since only 1 sqlinstance is accepted and it doesn't accept pipeline input.